website/i18n/zh/docusaurus-plugin-content-blog/2022/07/05/use-dataant-to-monitor-apisix.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "如何使用 DataAnt 监控 Apache APISIX"
+authors:
+  - name: "韩飞"
+    title: "Technical Writer"
+    url: "https://github.com/hf400159"
+    image_url: "https://github.com/hf400159.png"
+keywords: 
+- Apache APISIX
+- API 网关
+- Dataant
+description: 本文主要介绍如何通过 DataAnt Agent 将 API Gateway Apache APISIX 的指标数据上传到 DATAANT 监控系统。
+tags: [Authentication]
+---
+
+> 本文主要介绍如何通过 DataAnt Agent 将 API Gateway Apache APISIX 的指标数据上传到 DATAANT 监控系统。
+
+<!--truncate-->
+
+## 背景信息
+
+Apache APISIX 是一个开源的云原生 API 网关，作为 API 网关，它兼具动态、实时、高性能等特点，提供了负载均衡、动态上游、灰度发布、服务熔断、身份认证、可观测性等丰富的流量管理功能。你可以使用 Apache APISIX 来处理传统的南北向流量，以及服务间的东西向流量，也可以当做 K8s Ingress controller 来使用。得益于 APISIX 全动态的设计，可以随时进行配置更改并且均不需要重启服务。
+
+DataAnt 全栈云监控系统通过大数据和机器学习能够汇总 IaaS、PaaS 和 SaaS 层所有运维数据，给用户提供统一可视化界面。DataAnt 允许用户在相关监控数据源之间无缝快速地移动，而无需切换工具，更精细地了解到自己 IT 系统的状态。其提供的 DataAnt Agent 可以实时监控 APISIX 并将其监控数据上传到 DataAnt PaSS 平台，实现云端一站式监控。
+
+## 原理简介
+
+![principle](https://static.apiseven.com/2022/blog/0705/1.svg)
+
+1. 采集配置
+
+DataAnt Agent 首先会通过 `config.yaml` 的配置的 APISIX 项进行采集器初始化和注册。同一个 Agent 可以注册多个采集器。采集器收集 APISIX 暴露的指标后，对指标数据进行加密上传到 DataAnt Cloud。
+
+2. 数据可视化
+
+DataAnt Cloud 在接收到数据后，数据经过初步的监控信息补充和处理会存放到时序数据库中，之后可以通过 DataAnt 的 Dashboard 实时监控 APISIX。
+
+3. 告警通知
+
+数据还将通过消息分发到告警匹配处理，然后再进行通知聚合最终通过配置的通知方式将告警发送，即可以实时接收到 APISIX 的异常情况。
```

Comment:
Wrong order, need to fix.

<img width="857" alt="image" src="https://user-images.githubusercontent.com/36651058/178105818-5493279e-c9eb-46d5-9f4f-da00a1bd8cba.png">